website/docs/components/table/table/partials/guidelines/guidelines.md

```diff
@@ -221,3 +216,8 @@ This documentation currently covers only fundamental multi-select table interact
     type="video/mp4"
   />
 </video>
+
+!!! Callout
```

Comment:
Why was this moved below the content? I'd either expect it under the heading "Intended interaction" or before the video. Having it under the video is odd, as the video is the least important item in the hierarchy.